src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -99,6 +104,17 @@ public static Builder builder(Class<?> type) {
     return builder(ClassName.get(type));
   }
 
+  public static AnnotationSpec get(AnnotationMirror annotation) {
```

Comment:
Sure. Will move the code to the builder construct. Copied the get()-ty way from ClassName ... but you never have those static factory methods when there's a builder available. Correct?